src/main/resources/multiverse-core_en.properties

```diff
@@ -19,6 +19,17 @@ mv-core.clone.success=Cloned world '{world}'!
 # /mv confirm
 mv-core.confirm.description=Confirms dangerous commands before executing them.
 
+# /mv coordinates
+mv-core.coordinates.description=Simply sends your coordinates
+mv-core.coordinates.error.players-only=&cThis command can only be used by players
+mv-core.coordinates.error.multiverseWorldOnly=&cThis can only be used in multiverse worlds
```

Comment:
I think this 2 message can be generic error messages like `mv-core.commands.error.mutliverseworldonly`. Bcu seems like it will be used for other commands as well.

Reply:
Ooo, good point. I’ll swap it over